docs/edgeql/paths.rst

```diff
@@ -78,58 +137,75 @@ Starting from each user, the path below traverses all *incoming* links labeled
 
 .. code-block:: edgeql
 
-  select User.<author;
+    select User.<author;
+
+This query works, showing both the ``BlogPost`` and the ``Comment`` in the
+database. However, we can't impose a shape on it:
+
+.. code-block:: edgeql
+
+    select User.<author { text };
 
 As written, EdgeDB infers the *type* of this expression to be
-:eql:type:`BaseObject`, not ``BlogPost``. Why? Because in theory, there may be
-several links named ``author`` that point to ``User``.
+:eql:type:`BaseObject`. Why? Because in theory, there may be
+several links named ``author`` that point to ``User``. And there
```

Comment:
"...several links **from different object types** that point to ``User``, all named ``author``..."

to emphasize the reason not all links would have the `text` property

Reply:
Okay, took your wording in mostly verbatim: https://github.com/edgedb/edgedb/commit/8512b52ec6882d7fa67d73e99d0afd63e27f3ee6